File: 백준/Bronze/10214. Baseball/Baseball.py
import sys
input=lambda:sys.stdin.readline().rstrip()
for _ in range(int(input())):
    sum_y,sum_k=0,0
    for _ in range(9):
        y,k=map(int,input().split())
        sum_k+=k
        sum_y+=y
    print("Draw" if sum_y==sum_k else "Yonsei" if sum_y>sum_k else "Korea")

File: 백준/Bronze/10214. Baseball/Baseball.cc
#include <iostream>
using namespace std;
int main(void) {
    int t, k, y, tot_k, tot_y;
    cin >> t;
    for (int i = 0; i < t; i++) {
        tot_k = 0, tot_y = 0;
        for (int j = 0; j < 9; j++) {
            cin >> y >> k;
            tot_y += y;
            tot_k += k;
        }
        cout << (tot_k == tot_y ? "Draw" : (tot_y > tot_k ? "Yonsei" : "Korea")) << "\n";
    }
    return 0;
}